core/src/test/java/bisq/core/offer/OfferUtilTest.java

```diff
@@ -55,24 +55,24 @@ public void testGetAdjustedAmount() {
         }
 
         result = OfferUtil.getAdjustedAmount(
-                Coin.valueOf(1_000_000),
+                Coin.valueOf(100_000_000),
                 Price.valueOf("USD", 1000_0000),
-                20_000_000,
+                200_000_000,
                 1);
         Assert.assertEquals(
                 "Minimum allowed trade amount should not be adjusted.",
-                "0.01 BTC",
+                "1.00 BTC",
                 result.toFriendlyString()
         );
 
         result = OfferUtil.getAdjustedAmount(
-                Coin.valueOf(100_000),
+                Coin.valueOf(10_000_000),
                 Price.valueOf("USD", 1000_0000),
-                1_000_000,
+                20_000_000,
```

Comment:
Why is this 20x when others are 100x?

Reply:
Could it be related to 1 XMR = ~5200 ETH?

Reply:
Yes. Simly for consistency.